File: Architecture.Core/Entities/ProfAssetInfo.cs
using System;
using System.Collections.Generic;
using System.Text;
using Architecture.Core.Entities.Core;
using Architecture.Core.Entities.LU;

namespace Architecture.Core.Entities
{
   public class ProfAssetInfo : Auditable
    {
        public int AssetInfoId { get; set; }
        public int ProfileId { get; set; }
        public virtual ProfBasicInfo ProfBasicInfo { get; set; }
        public int? AssetTypeId { get; set; }
        public virtual AssetType AssetType { get; set; }
        public int NumberOfAsset { get; set; }
        public int EquivalentMoneyMax { get; set; }
        public int EquivalentMoneyMin { get; set; }
        public decimal MoneyAverage { get; set; }
        public int OwnerTypeId { get; set; }
        public virtual OwnerType OwnerType { get; set; }
        public decimal OwnershipPercentage { get; set; }
        public DateTime? OwnerFromDate { get; set; }
        public decimal RentAmount { get; set; }
        public decimal TaxAmount { get; set; }
        public decimal UseAblePercentage { get; set; }
        public string AnyRestrictionByGovt { get; set; }
        public string CityName { get; set; }
        public string Note { get; set; }

    }
}

File: Architecture.Web/ClientApp/src/app/Shared/Entity/ClientProfile/profAssetInfo.ts
import { Data } from '@angular/router';
import { Auditable } from '../Common/auditable';

export class profAssetInfo extends Auditable {
    public AssetInfoId: number;
    public profileId: number;
    //public virtual ProfBasicInfo ProfBasicInfo { get; set; }
    
    public  assetTypeId : number
    //public virtual AssetType AssetType { get; set; }
    public assetTypeName: string

    public numberOfAsset: number

    public equivalentMoneyMax: number

    public equivalentMoneyMin: number

    public moneyAverage: number

    public ownerTypeName: string

    public ownerTypeId: number
    //public virtual OwnerType OwnerType { get; set; }

    public ownershipPercentage: number

    public ownerFromDate: Date

    public rentAmount: number
    public taxAmount: number
    public useAblePercentage: number

    public anyRestrictionByGovt: string; 

    public cityName: string;
    
    public note: string; 
        
}


